<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>View</class>
  <widget class="QWidget" name="View">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>3D Viewer v2.0</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
        <widget class="QWidget" name="opengl_widget" native="true">
          <property name="minimumSize">
            <size>
              <width>800</width>
              <height>600</height>
            </size>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QWidget" name="control_panel" native="true">
          <property name="maximumSize">
            <size>
              <width>400</width>
              <height>16777215</height>
            </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
            <item>
              <widget class="QGroupBox" name="groupBox_file">
                <property name="title">
                  <string>Файл</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_file">
                  <item>
                    <widget class="QPushButton" name="pushButton_load_file">
                      <property name="text">
                        <string>Выбрать файл</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLabel" name="label_filename">
                      <property name="text">
                        <string>Файл не выбран</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLabel" name="label_file_info">
                      <property name="text">
                        <string>Вершин: 0, Рёбер: 0</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <widget class="QGroupBox" name="groupBox_transform">
                <property name="title">
                  <string>Трансформации</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_transform">
                  <item>
                    <widget class="QLabel" name="label_move">
                      <property name="text">
                        <string>Перемещение</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_move_x">
                      <item>
                        <widget class="QLabel" name="label_move_x">
                          <property name="text">
                            <string>X:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_move_x">
                          <property name="minimum">
                            <number>-100</number>
                          </property>
                          <property name="maximum">
                            <number>100</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_move_y">
                      <item>
                        <widget class="QLabel" name="label_move_y">
                          <property name="text">
                            <string>Y:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_move_y">
                          <property name="minimum">
                            <number>-100</number>
                          </property>
                          <property name="maximum">
                            <number>100</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_move_z">
                      <item>
                        <widget class="QLabel" name="label_move_z">
                          <property name="text">
                            <string>Z:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_move_z">
                          <property name="minimum">
                            <number>-100</number>
                          </property>
                          <property name="maximum">
                            <number>100</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <widget class="QLabel" name="label_rotate">
                      <property name="text">
                        <string>Поворот</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_rotate_x">
                      <item>
                        <widget class="QLabel" name="label_rotate_x">
                          <property name="text">
                            <string>X:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_rotate_x">
                          <property name="minimum">
                            <number>-180</number>
                          </property>
                          <property name="maximum">
                            <number>180</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_rotate_y">
                      <item>
                        <widget class="QLabel" name="label_rotate_y">
                          <property name="text">
                            <string>Y:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_rotate_y">
                          <property name="minimum">
                            <number>-180</number>
                          </property>
                          <property name="maximum">
                            <number>180</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_rotate_z">
                      <item>
                        <widget class="QLabel" name="label_rotate_z">
                          <property name="text">
                            <string>Z:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_rotate_z">
                          <property name="minimum">
                            <number>-180</number>
                          </property>
                          <property name="maximum">
                            <number>180</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                  <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_scale">
                      <item>
                        <widget class="QLabel" name="label_scale">
                          <property name="text">
                            <string>Масштаб:</string>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QSlider" name="horizontalSlider_scale">
                          <property name="minimum">
                            <number>1</number>
                          </property>
                          <property name="maximum">
                            <number>500</number>
                          </property>
                          <property name="value">
                            <number>100</number>
                          </property>
                          <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <spacer name="verticalSpacer">
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>20</width>
                    <height>40</height>
                  </size>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
